app/templates/profile.hbs

```diff
@@ -9,12 +9,12 @@
       </h1>
       <div class="section-menu hide-for-print">
         <ul class="menu">
-          <li><a href='#indicators'>Indicators</a></li>
-          <li><a href='#built-environment'>Built Environment</a></li>
-          <li><a href='#floodplain'>Floodplain</a></li>
-          <li><a href='#community-board'>Community Board</a></li>
-          <li><a href='#projects'>Projects</a></li>
-          <li><a href='#resources'>Resources</a></li>
+          <li><a href='#indicators' onclick={{action 'trackAnchorLink' 'Indicators'}}>Indicators</a></li>
```

Comment:
Not holding up this PR over this, just a general note/question: Should this really be necessary? Can't matomo automatically track when the host and path changes in the user's browser? I know that can be complicated by the fact that this is an SPA but I would think it could still hook into the browser History API somehow. It just seems like this approach is going to get messy if we have to create these little functions for every `<a>` that we care about across all of the apps.

Reply:
It can track using the browser history api, but I believe in my testing it was not correctly putting the anchor link on the URL, so it was just recording it as a page reload.